### PR TITLE
external contract method call support

### PR DESCRIPTION
**Pull request by RohitRanjangit**

### CODE REVIEW

File: src/passes/addressHandler/addressArgumentPusher.ts [hunk 1]
```diff
@@ -1,37 +0,0 @@
-import {
```
Comment (BitBaseBit) [inline src/passes/addressHandler/addressArgumentPusher.ts:1]: Why was this removed?

Comment (DomHenderson) [reply src/passes/addressHandler/addressArgumentPusher.ts:1]: Yeah, doing the transform for cross contract calling in the AST is a huge pain to do in a way that works nicely with other passes and doesn't have to be constantly designed around anytime we process functions. The addressHandler and its constituent passes have been deprecated for a while because of this, they were only kept in to throw a NotSupportedYet when they would have pushed an argument. Now that such calls are supported they're not needed.
